src/lib/pickles/plonk_checks/plonk_checks.ml

```diff
@@ -396,8 +396,8 @@ module Make (Shifted_value : Shifted_value.S) (Sc : Scalars.S) = struct
       in
       let generic =
         let open Vector in
-        let (l1 :: r1 :: o1 :: l2 :: r2 :: o2 :: _) = w0 in
-        [ e0 generic_selector; l1; r1; o1; l2; r2; o2 ]
+        let (l1 :: r1 :: o1 :: _) = w0 in
+        [ e0 generic_selector; l1; r1; o1 ]
```

Comment:
can you explain a bit more about that change? I'm confused as to why we're removing these

Reply:
It failed when I tried to remove everything, so I started bisecting to see what I could remove. Turned out that some of the test circuits dropped below the domain size as these were removed.